### DISCUSSION TITLE
Custom login page?

### DISCUSSION THREAD

**Discussion by infracritical** [formbricks/formbricks#4349] [General]
Is there a way to modify the login page?  I am interested in modifying the left-side column portion with advertisements and rotate them randomly.  Does Formbricks allow/permit this, and if so, how is it done?

**Comment by mattinannt** [comment]
Formbricks is open source under the AGPLv3 license. You can create a fork of the repository and make the changes you want to it as long as it only affects the code licensed under AGPLv3 and also complies with the license :-)
